examples/pipelines/filter-cc-25m/pipeline.py

```diff
@@ -30,22 +30,19 @@ def create_directory_if_not_exists(path):
     "license_location": "images_license+location",
     "license_type": "images_license+type",
     "webpage_url": "images_webpage+url",
+    "surt_url": "images_surt+url",
+    "top_level_domain": "images_top+level+domain",
 }
 
 load_from_hf_hub = ComponentOp(
     component_dir="components/load_from_hf_hub",
     arguments={
         "dataset_name": "fondant-ai/fondant-cc-25m",
         "column_name_mapping": load_component_column_mapping,
-        "n_rows_to_load": 1000,  # Here you can modify the number of images you want to download.
+        "n_rows_to_load": 100000,  # Here you can modify the number of images you want to download.
     },
 )
 
-# Filter mime type component
```

Comment:
Why are we removing this one again?

Reply:
its not used here but in `filter_pipeline.py`

Reply:
ohh ok thanks :)